standalone-metastore/src/main/scripts/ext/metastore.sh

```diff
@@ -17,17 +17,17 @@ THISSERVICE=metastore
 export SERVICE_LIST="${SERVICE_LIST}${THISSERVICE} "
 
 metastore() {
-  echo "$(timestamp): Starting Hive Metastore Server"
+  echo "$(timestamp): Starting Metastore Server"
   CLASS=org.apache.hadoop.hive.metastore.HiveMetaStore
   if $cygwin; then
-    HIVE_LIB=`cygpath -w "$HIVE_LIB"`
+    METASTORE_LIB=`cygpath -w "$METASTORE_LIB"`
   fi
-  JAR=${HIVE_LIB}/hive-metastore-*.jar
+  JAR=${METASTORE_LIB}/hive-standalone-metastore-*.jar
 
   # hadoop 20 or newer - skip the aux_jars option and hiveconf
 
   export HADOOP_CLIENT_OPTS=" -Dproc_metastore $HADOOP_CLIENT_OPTS "
-  export HADOOP_OPTS="$HIVE_METASTORE_HADOOP_OPTS $HADOOP_OPTS"
+  export HADOOP_OPTS="$METASTORE_HADOOP_OPTS $HADOOP_OPTS"
```

Comment:
changing environment variables used would break existing environments. Can we make these changes support old environment variable names as well?

Reply:
Fair point.  I want to make sure in the future people can set HIVE_LIB to one value and METASTORE_LIB to something else and not have them clash.  I'll follow the same path I did in the config file, which is use METASTORE_X if it's there, and if not fall back to HIVE_X.

Reply:
Thanks

Reply:
I may reverse course on this one.  Based on the discussion on the dev list, if we keep the start scripts for the metastore in hive proper as well as in the metastore then I don't think the metastore specific scripts need to be backwards compatible.  For users who are already using 'bin/hive --service metastore' setting HIVE_HOME etc. will continue to work.  For users using the new metastore_start script provided here it makes sense to set METASTORE_HOME etc. instead.

Reply:
I think it makes sense to have newly introduced scripts not needing to be backwards compatible since no one really used them before.

But since this script is called from hive — service metastore I think if we change this environment variable it will break some environments.

Eg: user has set some jvm options only for metastore using HIVE_METASTORE_HADOOP_OPTS now will not see them getting set after this change when using hive — service metastore

Reply:
This script isn't called by 'hive --service metastore'.  There is still a metastore.sh script in bin/ext/ for that, that will still respect HIVE_LIB etc.

Reply:
If this is a new script, it should be fine. But its weird that I don't see a bin/ext/metastore.sh in standalone-metastore branch. I only it in master branch. Thats the reason I thought it was moved to this new location.